### PR TITLE
feat: improve style extensibility

### PR DESCRIPTION
**Pull request by tommymorgan**
1. Separate variables from blueprint.scss so other apps can include just
the variables.
2. Compile, concatenate, and copy CSS to dist directory.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,10 +25,10 @@
     "node": ">=8.3.0"
   },
   "scripts": {
-    "build": "yarn build.styles && sl-scripts build && cp -a ./styles/. ./dist/styles",
-    "build.styles": "yarn build.bp && yarn build.tw",
-    "build.bp": "node-sass ./styles/blueprint.scss ./styles/blueprint.css --functions ./scripts/sass-inline-svg.js --importer node_modules/node-sass-package-importer/dist/cli.js",
-    "build.tw": "postcss ./styles/tailwind.css -o ./styles/index.css",
+    "build": "rm -rf build && sl-scripts build && yarn build.styles",
+    "build.styles": "yarn build.bp && yarn build.tw && mkdir -p dist && cat ./build/blueprint.css ./build/tailwind.css > ./dist/ui-kit.css && cp ./styles/_variables.scss ./dist/",
```
Comment (casserni) [inline package.json]: is there a reason to have this extra build folder?

Comment (tommymorgan) [reply package.json]: I created it to hold ephemeral files not intended for the dist or styles directories.  I think it belongs.

**Comment by casserni** [comment]
Also we need `blueprint.scss` to be exposed in the dist folder in its entirety, not just the the variables.

Reason being when those variables change we need to recompile the sass at runtime

**Comment by tommymorgan** [comment]
Blueprint.css is exposed in the dist directory via ui-kit.css.  I concatenated it with tailwind.css to reduce HTTP requests from the client.

**Comment by casserni** [comment]
Stale pr closing

**Pull request closed without merging**